pyproject.toml

```diff
@@ -0,0 +1,49 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "yattag"
+authors = [
+    {name = "Benjamin Le Forestier", email = "benjamin@leforestier.org"},
+]
+version = "1.15.2"
```

Comment:
You can avoid duplicating the version here by substituting this line with

```toml
dynamic = ["version"]
```

and adding the following at the bottom of the file

```toml
[tool.setuptools.dynamic]
version = {attr = "yattag.__version__"}
```